regimes/it/tags.go

```diff
@@ -0,0 +1,278 @@
+package it
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/tax"
+)
+
+// Tag keys determined from the "Natura" field from FatturaPA.
+const (
+	TagKeyExcluded                                cbc.Key = "excluded"
+	TagKeyNotSubject                              cbc.Key = "not-subject" // not used since 1 Jan 2021
+	TagKeyNotSubjectArticle7                      cbc.Key = "not-subject-article-7"
+	TagKeyNotSubjectOther                         cbc.Key = "not-subject-other"
+	TagKeyNotTaxable                              cbc.Key = "not-taxable" // not used since 1 Jan 2021
+	TagKeyNotTaxableExports                       cbc.Key = "not-taxable-exports"
+	TagKeyNotTaxableIntraCommunity                cbc.Key = "not-taxable-intra-community"
+	TagKeyNotTaxableSanMerino                     cbc.Key = "not-taxable-san-merino"
+	TagKeyNotTaxableExportSupplies                cbc.Key = "not-taxable-export-supplies"
+	TagKeyNotTaxableDeclarationOfIntent           cbc.Key = "not-taxable-declaration-of-intent"
+	TagKeyNotTaxableOther                         cbc.Key = "not-taxable-other"
+	TagKeyExempt                                  cbc.Key = "exempt"
+	TagKeyMarginRegime                            cbc.Key = "margin-regime"
+	TagKeyReverseCharge                           cbc.Key = "reverse-charge"
```

Comment:
Do you see some of these moving to `common` eventually, so other regimes like `es` can specify reverse charges using tags?

Reply:
Great point, but no, not in this case as the tags are very specific for Italy. The `TagKeyReverseCharge` is actually not allowed to be used any more, one of the other more descriptive codes needs to be used instead. While it is confusing, the `reverse-charge` scheme key should be used alongside the individual line tags.

Reply:
Oh that's right, I forgot the generic `natura`s gave way to more specific versions.